scripts/cli_options.py

```diff
@@ -1,13 +1,71 @@
 #! /usr/bin/env python
 from __future__ import absolute_import, print_function
 import random
+from optparse import OptionParser, OptionValueError
+
+import jmclient.support
 
 """This exists as a separate module for two reasons:
 to reduce clutter in main scripts, and (TODO) refactor out
 options which are common to more than one script in a base class.
 """
 
-from optparse import OptionParser
+order_choose_algorithms = (
+    'random_under_max_order_choose', 'cheapest_order_choose',
+    'weighted_order_choose')
+
+
+def add_common_options(parser):
+    parser.add_option(
+        '-f',
+        '--txfee',
+        action='store',
+        type='int',
+        dest='txfee',
+        default=-1,
+        help='number of satoshis per participant to use as the initial estimate '
+        'for the total transaction fee, default=dynamically estimated, note that this is adjusted '
+        'based on the estimated fee calculated after tx construction, based on '
+        'policy set in joinmarket.cfg.')
+    parser.add_option('--fast',
+                      action='store_true',
+                      dest='fastsync',
+                      default=False,
+                      help=('choose to do fast wallet sync, only for Core and '
+                            'only for previously synced wallet'))
+    parser.add_option(
+        '-x',
+        '--maxcjfee',
+        type='float',
+        dest='maxcjfee',
+        nargs=2,
+        default=(0.01, 10000),
+        help='maximum coinjoin fee and bitcoin value the tumbler is '
```

Comment:
Maybe update that this limit is not only relevant for the tumbler. I tested it successfully with sendpayment as well.